spec/components/organisation/school_overview_component_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'rails_helper'
+
+RSpec.describe Organisation::SchoolOverviewComponent, type: :component do
+  let!(:inline_component) { render_inline(described_class.new(organisation: organisation)) }
+
+  context 'when organisation is a SchoolGroup' do
+    let(:organisation) { build(:school_group) }
+
+    it 'does not render the school overview component' do
+      expect(rendered_component).to eql('')
```

Comment:
This reads better with:

`expect(rendered_component).to be_blank`

Reply:
Indeed it does, I'd tried `to be nil` but it returns the empty string. Thanks 👍